apps/sms/views/inbox/js/inbox.js

```diff
@@ -983,6 +975,8 @@ var InboxView = {
   onDraftSaved: function inbox_onDraftSaved(draft) {
     var threadToUpdate = draft.threadId ? Threads.get(draft.threadId) : draft;
     this.updateThread(threadToUpdate);
+
+    this.showDraftSavedBanner();
```

Comment:
This is one more thing I'm not sure of. The problem is that I've moved draft manipulation code to ```beforeLeave``` so I don't have access to that info (saved or not) when I navigate to Inbox. With this we always show banner when draft is saved even if view is not shown (it's shown for 5 seconds), I don't know if it's too bad for you :)

To simulate previous behavior I can leave ```showDraftSavedBanner``` in ```beforeEnter``` as previously and in ```onDraftSaved``` have something like
```
this.draftHasBeenSavedRecently = true;
setTimeout(() => this.draftHasBeenSavedRecently = false, 2000);
```

And if in ```InboxView.beforeEnter``` this variable is true, we show the banner. Or maybe you have better idea?